handbook/intro.md

```diff
@@ -17,6 +33,8 @@ A GitHub organization is a shared account where all organization members can col
 5. [NSAPH Data Processing](https://github.com/NSAPH-Data-Processing) - Captures the data pipelines for data transformations and aggregation;
 6. [NSAPH](https://github.com/NSAPH) - Gathers past projects but will continue to be used for miscellaneous projects that don't belong in the previous categories.
```

Comment:
This duplicates 2 (it shows the main NSAPH GitHub twice); can we consolidate them?